lib/Pipeline/ContainerSet.cpp

```diff
@@ -83,8 +83,13 @@ llvm::Error ContainerSet::store(const revng::DirectoryPath &Directory) const {
     if (Container == nullptr)
       continue;
 
+    if (not Container->isDirty())
+      continue;
```

Comment:
Please verify that this work and that we're actually writing less stuff.

Reply:
I've already checked, less things are written.